test/common/components/LoginModal.unit.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import LoginModal from '../../../src/js/common/components/LoginModal';
+
+describe('<LoginModal>', () => {
```

Comment:
Still need to test signing in, but since `handleLogin` is called from login helpers, I'm not sure how to do that yet.